src/domain/program.ts

```diff
@@ -165,10 +174,16 @@ export class ProgramManager
 
   async add(newProgram: AddProgram): Promise<Program> {
     try {
+      console.log('newProgram', newProgram)
```

Comment:
I'll try to keep the logs as minimal as possible to limit the console output. We could add an eslint warning to keep them at a reasonable level.